README.md

```diff
@@ -555,26 +557,47 @@ These are the steps for manual set-up of certificates:
    $ KUBCONFIG="<<your cluster kubeconfig path>> PATH="$PATH:$PWD/_work/bin" ./test/setup-ca-kubernetes.sh
 ```
 
-- **Deploy the driver to Kubernetes using DeviceMode:LVM**
+- **Deploy the driver to Kubernetes**
 
-```sh
-    $ sed -e 's/PMEM_REGISTRY/<your registry>/' deploy/kubernetes-<kubernetes version>/pmem-csi-lvm.yaml | kubectl create -f -
-```
+The `deploy/kubernetes-<kubernetes version>` directory contains
+`pmem-csi*.yaml` files which can be used to deploy the driver on that
+Kubernetes version. The files in the directory with the highest
+Kubernetes version might also work for more recent Kubernetes
+releases.
+
+For each Kubernetes version, four different deployment variants are provided:
+
+   - `direct` or `lvm`: one uses DeviceMode:Direct, the other DeviceMode:LVM
+   - `testing`: the variants with `testing` in the name enable debugging
+     features and shouldn't be used in production
+
+All of these files use `PMEM_REGISTRY` as placeholder for the actual Docker registry
+that contains the PMEM-CSI image. This needs to be replaced as part of the command which
+deploys the to Kubernetes.
```

Comment:
I think _**deploys the to Kubernetes**_ is missing something.

Is it:
* deploys PMEM-CSI to Kubernetes?